File: Nível 3/1049 - Animal/main.lua
io.write("A:")
informacao1 = io.read()
io.write("B:")
informacao2 = io.read()
io.write("C:")
informacao3 = io.read()

if informacao1 == "vertebrado" then
    if informacao2 == "ave" then
        if informacao3 == "carnivoro" then
            print("aguia")
        elseif informacao3 == "onivoro" then
            print("pomba")
        else
            print("erro")
        end
    elseif informacao2 == "mamifero" then
        if informacao3 == "onivoro" then
            print("homem")
        elseif informacao3 == "herbivoro" then
            print("vaca")
        else
            print("erro")
        end
    else
        print("erro")
    end
elseif informacao1 == "invertebrado" then
    if informacao2 == "inseto" then
        if informacao3 == "hematofago" then
            print("pulga")
        elseif informacao3 == "herbivoro" then
            print("lagarta")
        else
            print("erro")
        end
    elseif informacao2 == "anelideo" then
        if informacao3 == "hematofago" then
            print("sanguessuga")
        elseif informacao3 == "onivoro" then
            print("minhoca")
        else
            print("erro")
        end
    else
        print("erro")
    end
else
    print("erro")
end

File: Nível 3/1049 - Animal/main.py
informacao1 = input("A:")
informacao2 = input("B:")
informacao3 = input("C:")

if informacao1 == "vertebrado":

    if informacao2 == "ave":
        if informacao3 == "carnivoro":
            print("aguia")
        elif informacao3 == "onivoro":
            print("pomba")
        else:
            print("erro")

    elif informacao2 == "mamifero":
        if informacao3 == "onivoro":
            print("homem")
        elif informacao3 == "herbivoro":
            print("vaca")
        else:
            print("erro")

if informacao1 == "invertebrado":

    if informacao2 == "inseto":
        if informacao3 == "hematofago":
            print("pulga")
        elif informacao3 == "herbivoro":
            print("lagarta")
        else:
            print("erro")
    elif informacao2 == "anelideo":
        if informacao3 == "hematofago":
            print("sanguessuga")
        elif informacao3 == "onivoro":
            print("minhoca")
        else:
            print("erro")